mkdocs_csc/disable_ga.html

```diff
@@ -0,0 +1,67 @@
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ config.google_analytics[0] }}"></script>
+    <script>
+        function getCookie(cname) {
+            var name = cname + "=";
+            var decodedCookie = decodeURIComponent(document.cookie);
+            var ca = decodedCookie.split(';');
+            for(var i = 0; i <ca.length; i++) {
+                var c = ca[i];
+                while (c.charAt(0) == ' ') {
+                    c = c.substring(1);
+                }
+                if (c.indexOf(name) == 0) {
+                    return c.substring(name.length, c.length);
+                }
+            }
+            return "";
+        }
+var consent=getCookie("cookieconsent_status")
+if(consent == "" || consent != "allow" ){
```

Comment:
Not that important but `consent == ""` looks redundant and can be removed.

Reply:
fixed